### PR TITLE
Added a site template to use the material design theme

### LINKED ISSUE #60: Give a template for including material design
A template index.html file alongwith all the required files in separate folders.

I am working on this right now. And will send you a pull request soon.

### PR DESCRIPTION
**Pull request by icyflame**
The current `demo/` folder is too cluttered, since it has to show case all the elements that are there inside this theme.

The `template/index.html` file is a basic skeleton for any site that may want to use this theme.

Fixes #60

### CODE REVIEW

File: template/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<html>
+
+<head>
+
+	<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
```
Comment (FezVrasta) [inline template/index.html]: Please don't set the protocol, use `href="//maxcdn...."`. Thanks!

**Comment by FezVrasta** [comment]
Thank you! Please see my note above.

Probably I'll need to write a Grunt/Gulp task to compile everything and copy everithing in the right folders...

**Comment by icyflame** [comment]
I have removed the protocol from the `link` tag.

And by copying everything, do you mean that you need to write a grunt task so that whenever the project is built, latest versions of all the four files (material and ripples) get copied into the `template/material/` folder?

**Comment by FezVrasta** [comment]
same thing for jquery ^_^"

**Comment by icyflame** [comment]
Fixed. Sorry, I was not paying attention, and adding http became a practise.

**Pull request merged**